docs/tutorials/podman-install-certificate-authority.md

```diff
@@ -0,0 +1,75 @@
+![PODMAN logo](../../logo/podman-logo-source.svg)
+
+Install Certificate Authority
+=============================
+
+Organizations may create their own local certificate authority (CA) or acquire one from a third party.  This may mean more than one certificate such as one or more intermediate certificates and a root certificat for example.  In any case, there is a need to add the certificate authority (CA) certificate(s) such that it can be employed for various use cases.
```

Comment:
```suggestion
Organizations may create their own local certificate authority (CA) or acquire one from a third party.  This may mean more than one certificate, such as one or more intermediate certificates and a root certificate, for example.  In any case, it is necessary to add the certificate authority (CA) certificate(s) so that it can be employed for various use cases.
```